zulipterminal/core.py

```diff
@@ -433,13 +433,31 @@ def show_typing_notification(self) -> None:
 
         # Until conversation becomes "inactive" like when a `stop` event is sent
         while self.active_conversation_info:
-            sender_name = self.active_conversation_info["sender_name"]
-            self.view.set_footer_text(
-                [
-                    ("footer_contrast", " " + sender_name + " "),
-                    " is typing" + next(dots),
-                ]
+            active_conversation_info = ", ".join(
+                map(str, self.active_conversation_info)
```

Comment:
If these are already strings - try changing that `Any` I mention elsewhere - then this is misleading, since we don't need to apply that mapping.